fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/SelectItemSubQuery.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2019
+ * (C) Copyright IBM Corp. 2019,2021
```

Comment:
probably doesn't need copyright

Reply:
Also this class dead ends at  addSubQuery... is this actually used or useful?

Reply:
There are some old classes here no longer required. Will look at purging them where I see it is safe to do so.